packages/fscomponents/src/components/ZoomCarousel/types.ts

```diff
@@ -44,4 +44,18 @@ export interface ZoomCarouselProps {
   thumbnailStyle?: any;
   thumbnailContainerStyle?: any;
   contentContainerStyle?: StyleProp<ViewStyle>;
+
+  /**
+   * Boolean to turn on and off the page indicator (dots)
+   *
+   * @example true
+   */
+  hidePageIndicator?: boolean;
+
+  /**
+   * The styling used if fill container is set to true (that makes all carousel elememts expand)
+   *
+   * @example {{height: '50%'}}
+   */
+  fillContainerStyle?: StyleProp<ViewStyle>;
```

Comment:
This doesn't appear to be utilized anywhere in the ZoomCarousel code.

Reply:
Resolved here: https://github.com/brandingbrand/flagship/pull/1087/files#diff-badb4df4ea5cc3f25061a62fb1d31be9R309